src/lib/Microsoft.Health.Events/Telemetry/Exceptions/EventHubConfigurationExceptionTelemetryProcessor.cs

```diff
@@ -1,67 +0,0 @@
-﻿// -------------------------------------------------------------------------------------------------
```

Comment:
Removing this class as the list of handledexceptions are not really handleable based on the api's contract, in that it cannot be continued upon. In any case, the caller to this wasn't utilizing the return value of HandleException but was just logging the associated error metric, so updated that to just call the log api instead.
When we do identify exceptions that are handledable we should add those in here instead.